python_modules/dagster-webserver/dagster_webserver/cli.py

```diff
@@ -26,35 +27,32 @@
 from .version import __version__
 
 
-def create_dagit_cli():
-    return dagit
+def create_dagster_webserver_cli():
+    return dagster_webserver
 
 
-DEFAULT_DAGIT_HOST = "127.0.0.1"
-DEFAULT_DAGIT_PORT = 3000
+DEFAULT_WEBSERVER_HOST = "127.0.0.1"
+DEFAULT_WEBSERVER_PORT = 3000
 
 DEFAULT_DB_STATEMENT_TIMEOUT = 15000  # 15 sec
 DEFAULT_POOL_RECYCLE = 3600  # 1 hr
 
 
 @click.command(
-    name="dagit",
+    name="dagster-webserver",
     help=(
-        "Run dagit. Loads a repository or pipeline/job.\n\n{warning}".format(
+        "Run dagster-webserver. Loads a repository or pipeline/job.\n\n{warning}".format(
             warning=WORKSPACE_TARGET_WARNING
         )
         + (
-            "\n\nExamples:"
-            "\n\n1. dagit (works if .{default_filename} exists)"
-            "\n\n2. dagit -w path/to/{default_filename}"
-            "\n\n3. dagit -f path/to/file.py"
-            "\n\n4. dagit -f path/to/file.py -d path/to/working_directory"
-            "\n\n5. dagit -m some_module"
-            "\n\n6. dagit -f path/to/file.py -a define_repo"
-            "\n\n7. dagit -m some_module -a define_repo"
-            "\n\n8. dagit -p 3333"
-            "\n\nOptions can also provide arguments via environment variables prefixed with DAGIT"
-            "\n\nFor example, DAGIT_PORT=3333 dagit"
+            "\n\nExamples:\n\n1. dagster-webserver (works if .{default_filename} exists)\n\n2."
+            " dagster-webserver -w path/to/{default_filename}\n\n3. dagster-webserver -f"
+            " path/to/file.py\n\n4. dagster-webserver -f path/to/file.py -d"
+            " path/to/working_directory\n\n5. dagster-webserver -m some_module\n\n6."
+            " dagster-webserver -f path/to/file.py -a define_repo\n\n7. dagster-webserver -m"
+            " some_module -a define_repo\n\n8. dagster-webserver -p 3333\n\nOptions can also"
+            " provide arguments via environment variables prefixed with DAGSTER_WEBSERVER\n\nFor"
+            " example, DAGSTER_WEBSERVER_PORT=3333 dagster-webserver"
```

Comment:
nit: this seems much harder to edit now. We should probably drop the "repo" language

Reply:
Yeah I'm actually not even sure how this reformat happened. I've now changed it to use a """ string with `textwrap.dedent`, which improves readability/editability of original.